libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -35,6 +39,50 @@ const getIcon = (content) => {
   return icons.company;
 };
 
+const showTooltip = (element, message, time = 5000) => {
+  element.setAttribute('data-pep-dismissal-tooltip', message);
+  const cleanup = () => {
+    element.removeAttribute('data-pep-dismissal-tooltip');
+  };
+  const id = setTimeout(() => {
```

Comment:
What does `id` represent in this case? It doesn't seem to reflect its purpose at first sight

Reply:
It's the unique positive integer value returned by `setTimeout`. I've changed the name to be more expressive.